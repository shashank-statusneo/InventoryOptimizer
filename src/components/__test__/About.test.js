import { render, screen, cleanup } from '@testing-library/react'
import About from '../About'
import renderer from 'react-test-renderer'

afterEach(() => {
    cleanup()
})

test('test', () => {
    expect(true).toBe(true)
})

test('render react <About />', () => {
    render(<About />)
    const linkElement = screen.getByText(/About/i)
    expect(linkElement).toBeInTheDocument()
})

it('snapshot <About />', () => {
    const tree = renderer.create(<About />).toJSON()
    expect(tree).toMatchSnapshot()
})
