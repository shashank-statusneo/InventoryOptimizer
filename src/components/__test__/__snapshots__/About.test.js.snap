// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot <About /> 1`] = `
<h1>
  About
</h1>
`;
